web/obs/src/components/chat/AuthorChip.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <yt-live-chat-author-chip>
+    <span
+      id="author-name"
+      dir="auto"
+      class="style-scope yt-live-chat-author-chip"
+      :class="{ member: isInMemberMessage }"
+      :type="authorTypeText">
+      <!-- why there was <template> before? -->
+      <span>{{ authorName }}</span>
+      <!-- 这里是已验证勋章 -->
+      <span
+        id="chip-badges"
+        class="style-scope yt-live-chat-author-chip"></span>
+    </span>
+    <span id="chat-badges" class="style-scope yt-live-chat-author-chip">
+      <AuthorBadge
+        v-if="isInMemberMessage"
+        class="style-scope yt-live-chat-author-chip"
+        :isAdmin="false"
+        :privilegeType="privilegeType" />
```

Comment:
我更偏向于使用 kebab-case 来写组件的属性名
```suggestion
        :privilege-type="privilegeType" />
```